src/lsstseries/utils/column_mapper/column_mapper.py

```diff
@@ -0,0 +1,200 @@
+class ColumnMapper:
+    """Maps columns from a given dataset into known ensemble column"""
+
+    def __init__(
+        self,
+        id_col=None,
+        time_col=None,
+        flux_col=None,
+        err_col=None,
+        band_col=None,
+        provenance_col=None,
+        nobs_total_col=None,
+        nobs_band_cols=None,
+    ):
+        """
+
+        Parameters
+        ----------
+        id_col: 'str', optional
+            Identifies which column contains the Object IDs
+        time_col: 'str', optional
+            Identifies which column contains the time information
+        flux_col: 'str', optional
+            Identifies which column contains the flux/magnitude information
+        err_col: 'str', optional
+            Identifies which column contains the flux/mag error information
+        band_col: 'str', optional
+            Identifies which column contains the band information
+        nobs_col: list of 'str', optional
+            Identifies which columns contain number of observations for each
+            band, if available in the input object file
+        nobs_tot_col: 'str', optional
+            Identifies which column contains the total number of observations,
+            if available in the input object file
+        provenance_col: 'str', optional
```

Comment:
nit: `provenance_col` is out of order compared to the arguments in the function definition.